src/NetSparkle/AppCastHandlers/XMLAppCast.cs

```diff
@@ -10,7 +10,7 @@
 namespace NetSparkleUpdater.AppCastHandlers
 {
     /// <summary>
-    /// An XML-based appcast document downloader and handler
+    /// An XML-based app-cast document downloader and handler
```

Comment:
I made an arbitrary decision a while back that "app cast" was 2 words instead of one. so, nitpick, but the `-` should go :)

Reply:
Arbitrary decisions - lol - got it - change made across the codebase in my fork.

Reply:
My initial purpose to refactor this code was because I envisaged that users who wished to have a custom filtering process might want to use it.  

I consider it worthless refactoring at this point though. 

This is because in GetAvailableUpdates() the call to filter is made _first_, and then secondly the GetAvailableUpdates() method takes the resulting version/filtered list and runs is over the IsMatchingUpdate() algorithm anyway - so there is presently no need for end developers to implement this.  The results - even with filtering present - are _always_ passed through the IsMatchingUpdate() method.

I can put it back how it was if you prefer.